src/App.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+import Quotes from './components/quotes';
+import { Suspense } from 'react';
+import { NavLink, Link } from 'react-router-dom';
+import { Route } from 'react-router-dom';
+import { Routes } from 'react-router-dom';
+
+
```

Comment:
- [Optional] Please try to separate your code into different levels of commits. Remember your commits are going to help you look back and debug your code. It's a good idea to commit after every visible change.